packages/ra-input-rich-text/src/index.ts

```diff
@@ -0,0 +1 @@
+export * from './RichTextInput';
```

Comment:
Breaking change as we now have a named export. However, we plan to replace this input with the tiptap version